test/box/sequence.result

```diff
@@ -1115,7 +1115,8 @@ s.index.pk.sequence_id == nil
 ...
 s:insert{nil, 'x'} -- error
 ---
-- error: 'Tuple field 1 type does not match one required by operation: expected unsigned'
+- error: 'Tuple field 1 type does not match one required by operation: expected unsigned,
+    got any'
```

Comment:
`expected unsigned, got any`? must be `got nil`.

Reply:
Now string representation of type is taken from `field_type_strs` and it has only next values: `any`, `unsigned`, `string`, `number`, `double`, `integer`, `boolean`, `varbinary`, `scalar`, `decimal`, `uuid`, `array`, `map`.

Reply:
It should print type of message pack value, not field type: field type for particular value is not defined.

Reply:
fix